nupic/frameworks/opf/metrics.py

```diff
@@ -484,12 +491,51 @@ def _compute(self):
     return self.aggregateError
 
 
+class MetricNegLL(AggregateMetric):
```

Comment:
I would prefer a longer but easier to understand name, such as `MetricNegLogLikelihood`